standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -92,10 +92,9 @@ object StandardizationJob {
     })
   }
 
-  private def obtainSparkSession(cmd: CmdConfig): SparkSession = {
+  private def obtainSparkSession(): SparkSession = {
     val spark = SparkSession.builder()
-      .appName(s"Standardisation ${cmd.datasetName} ${cmd.datasetVersion} ${cmd.reportDate} ${cmd.reportVersion}")
-      .config("spark.sql.codegen.wholeStage", false) //disable whole stage code gen - the plan is too long
+      .appName(s"Standardisation")
```

Comment:
Why dataset details are removed from the app name?
(also, can remove 's' before the quotes as well)

Reply:
Yeah so the reason is simple. I needed spark session initialized in CmdConfig in order to use its Hadoop Configuration. 
So I believe that the ability to run cluster mode is higher priority than automatically creating nice app name. Application name can be modified during spark submit anyway.

Will remove s